examples/tutorial_examples/add_custom_bundle.py

```diff
@@ -0,0 +1,222 @@
+"""
+=============================
+Adding new bundles into pyAFQ (SLF 1/2/3 Example)
+=============================
+
+pyAFQ is designed to be customizable and extensible. This example shows how you
+can customize it to define a new bundle based on a definition of waypoint and
+endpoint ROIs of your design.
+
+In this case, we add the optic radiations, based on work by Caffara et al. [1]_,
```

Comment:
This paragraph (and much of the rest of the text) would have to be rewritten. I assume you are planning to do this after the code is running?